docs/pages/deploy-a-cluster/helm-deployments/azure.mdx

```diff
@@ -0,0 +1,325 @@
+highAvailability:
+  # Number of replicas to configure
+  replicaCount: 2
+  certManager:
+    # Enable cert-manager support to get TLS certificates
+    enabled: true
+    # Name of the cert-manager GlobalIssuer or Issuer to use
+    issuerName: letsencrypt-production
+    issuerKind: GlobalIssuer
+# If you are running Kubernetes 1.23 or above, disable PodSecurityPolicies
+podSecurityPolicy:
+  enabled: false
+# Indicate that this is a Teleport Enterprise deployment
+enterprise: true
+EOF
+```
+
+</ScopedBlock>
+
+Install the chart with the values from your `azure-values.yaml` file using this command:
+
+```code
+$ helm install <Var name="teleport-release" /> teleport/teleport-cluster \
+    --create-namespace --namespace <Var name="teleport-ns" /> \
+    --values azure-values.yaml
+```
+
+<Admonition type="note">
+  You cannot change the `clusterName` after the cluster is configured, so make sure you choose wisely. We recommend using the fully-qualified domain name that you'll use for external access to your Teleport cluster.
+</Admonition>
+
+Once the chart is installed, you can use `kubectl` commands to view the deployment:
+
+```code
+$ kubectl --namespace teleport get all
+
+NAME                                 READY   STATUS    RESTARTS   AGE
+pod/teleport-auth-57989d4cb-4q2ds    1/1     Running   0          22h
+pod/teleport-auth-57989d4cb-rtrzn    1/1     Running   0          22h
+pod/teleport-proxy-c6bf55cfc-w96d2   1/1     Running   0          22h
+pod/teleport-proxy-c6bf55cfc-z256w   1/1     Running   0          22h
+
+NAME                        TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)                                                                     AGE
+service/teleport            LoadBalancer   10.40.11.180   34.138.177.11   443:30258/TCP,3023:31802/TCP,3026:32182/TCP,3024:30101/TCP,3036:30302/TCP   22h
+service/teleport-auth       ClusterIP      10.40.8.251    <none>          3025/TCP,3026/TCP                                                           22h
+service/teleport-auth-v13   ClusterIP      None           <none>          <none>                                                                      22h
+service/teleport-auth-v14   ClusterIP      None           <none>          <none>                                                                      22h
+
+NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/teleport-auth    2/2     2            2           22h
+deployment.apps/teleport-proxy   2/2     2            2           22h
+
+NAME                                       DESIRED   CURRENT   READY   AGE
+replicaset.apps/teleport-auth-57989d4cb    2         2         2       22h
+replicaset.apps/teleport-proxy-c6bf55cfc   2         2         2       22h
+```
+
+## Step 4/5. Set up DNS
+
+We'll now set up DNS `A` records for `teleport.example.com` and `*.teleport.example.com`.
```

Comment:
I'd add something about how this assumes that you're using a domain configured with Azure DNS, and to add the records in your own provider if you're not.

Reply:
519d9fcda1